docs/Server/API/V1/relay.md

```diff
@@ -0,0 +1,143 @@
+# `POST /api/v1/relay/<uri>`
+
+This API creates a dataset resource by reading data from a Relay server in two
+steps. The provided URI represents a "manifest file" stored on the Relay server,
+a JSON file (`application/json` MIME format) defining the original tarball name,
+the MD5 hash value, optional metadata key/value pairs for the dataset, and a
+URI from which the Pbench Server expects to `GET` an `application/octet-stream`
+of a file created by `tar` and compressed with the `xz` program.
+
+Primarily this is expected to be a native Pbench Agent tarball with a specific
+structure; however with the `server.archiveonly` metadata key the Pbench Server
+can be used to archive and manage metadata for any tarball passed through a
+Relay server.
+
+## URI parameters
+
+`<uri>` string \
+The Relay server URI of the tarball's manifest `application/json` file. This
+must provide the following JSON keys:
+
+For example,
+
+```json
+{
+    "uri": "https://relay.example.com/52adfdd3dbf2a87ed6c1c41a1ce278290064b0455f585149b3dadbe5a0b62f44",
+    "md5": "22a4bc5748b920c6ce271eb68f08d91c".
+    "name": "fio_rw_2018.02.01T22.40.57.tar.xz".
```

Comment:
These lines appear to end in periods where they should have commas.

Reply:
I really can't do one-finger typing very well, and some of the key labels are wearing out. 😆